pymc/tests/test_distributions_timeseries.py

```diff
@@ -17,15 +17,33 @@
 
 from pymc.aesaraf import floatX
 from pymc.distributions.continuous import Flat, Normal
-from pymc.distributions.timeseries import AR, AR1, GARCH11, EulerMaruyama
+from pymc.distributions.timeseries import (
+    AR,
+    AR1,
+    GARCH11,
+    EulerMaruyama,
+    gaussianrandomwalk,
+)
 from pymc.model import Model
 from pymc.sampling import sample, sample_posterior_predictive
 from pymc.tests.helpers import select_by_precision
 
 # pytestmark = pytest.mark.usefixtures("seeded_test")
-pytestmark = pytest.mark.xfail(reason="Timeseries not refactored")
 
 
+def test_grw_rv_op():
+    """Basic test for GRW RV op"""
+    init = 1
+    mu = 3
+    sd = 0.0000001
+    size = 4
+
+    grw = gaussianrandomwalk(mu, sd, init, size).eval()
```

Comment:
Here is direct link to testing exception where the size argument ends up in the wrong place. Any advice would be great!

https://github.com/pymc-devs/pymc/runs/4710136699?check_suite_focus=true#step:7:2295

Reply:
I think size is a keyword only argument in RandomVariables

Reply:
That was it thank you